NineChronicles.Headless.Executable/Program.cs

```diff
@@ -218,10 +218,33 @@ static void ConfigureSentryOptions(SentryOptions o)
 
             Log.Logger = loggerConf.CreateLogger();
 
-            var tasks = new List<Task>();
+            if (appProtocolVersionToken is null)
+            {
+                throw new CommandExitedException(
+                    "--app-protocol-version must be present.",
+                    -1
+                );
+            }
+
+            if (genesisBlockPath is null)
+            {
+                throw new CommandExitedException(
+                    "--genesis-block-path must be present.",
+                    -1
+                );
+            }
```

Comment:
This comment has same intention with the comment about `appProtocolVersionToken`.

Reply:
Added. https://github.com/planetarium/NineChronicles.Headless/pull/481/commits/19b87259e7df2b0d4f2d676410c46d37d4fa60b4